### PR TITLE
php::globals: support Ubuntu 20.04 that ships php 7.4

### PR DESCRIPTION
**Pull request by simondeziel**
Replaces: #580

### CODE REVIEW

**Review by igalic — APPROVED** [review]

**Comment by pcfens** [comment]
I was able to confirm that this works - what needs to happen to go ahead and get this merged?

**Comment by kenyon** [comment]
We need to also add Ubuntu 20.04 to metadata.json, and get tests running for Ubuntu 20.04.

**Comment by bastelfreak** [comment]
hey @simondeziel, thanks for the PR. Can you take a look at the .sync.yml / .travis.yml and add Ubuntu 20.04 there as well? (just copy 'n paste the 18.04 block is probably fine). This will enable acceptance tests for this platform.

File: .travis.yml [hunk 1]
```diff
@@ -44,6 +44,16 @@ jobs:
       env: BEAKER_PUPPET_COLLECTION=puppet6 BEAKER_setfile=centos7-64 CHECK=beaker
       script: bundle exec rspec spec/acceptance/php56_spec.rb
       services: docker
+    - rvm: 2.5.3
+      bundler_args: --without development release
+      env: BEAKER_PUPPET_COLLECTION=puppet5 BEAKER_setfile=ubuntu2004-64 CHECK=beaker
+      script: bundle exec rspec spec/acceptance/php_spec.rb
+      services: docker
+    - rvm: 2.5.3
```
Comment (bastelfreak) [inline .travis.yml:52]: sorry for all the trouble :(
Puppet Inc. does not support Puppet 5 on Ubuntu 20.04. Can you remove this entry?

**Comment by kenyon** [comment]
I removed the failing puppet5 beaker test since puppet5 is not supported on Ubuntu 20.04: https://tickets.puppetlabs.com/browse/CPR-737

**Review by bastelfreak — APPROVED** [review]

**Comment by simondeziel** [comment]
Thank you both @bastelfreak and @kenyon. It looks like the last failure is due to a glitch preventing the installation of puppetlabs-apt module in the 16.04 env. The same test worked in previous updates of this PR. Could you either force a test re-run or maybe merge it as is? Thanks again.

**Comment by bastelfreak** [comment]
I restarted the failed job

**Pull request merged**